addons/l10n_ug/views/report_invoice.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="report_invoice_document" inherit_id="account.report_invoice_document">
+        <xpath expr="//div[@id='informations']" position="before">
+            <div t-if="o.company_id.country_code == 'UG'">
+                <t t-set="forced_vat" t-value="o.company_id.vat"/>
```

Comment:
Is this something specific to UG?
Shouldn't the company's VAT number already be in the header/footer anyways?

Reply:
Indeed! I had never notice it was there. My bad!